screens/AppNavigator/screens/TransactionsScreen/screens/TransactionDetailScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import tailwind from 'tailwind-rn'
+import { translate } from '../../../../../translations'
+import { View, Linking } from 'react-native'
+import { Text } from '../../../../../components'
+import { StackScreenProps } from '@react-navigation/stack'
+import { TransactionsParamList } from '../TransactionsNavigator'
+import { Ionicons } from '@expo/vector-icons'
+
+// FIXME(@ivan-zynesis): envvar
+const EXPLORER_BASE_URL = 'https://explorer.defichain.io'
```

Comment:
Should this one be updated based on network?

Reply:
Yeah, thus the FIXME is there. As we are working on new explorer, so the link here isnt really finalized (for every env).